src/Features/Blockcore.Features.Wallet/Api/Controllers/WalletController.cs

```diff
@@ -698,13 +699,18 @@ public IActionResult BuildTransaction([FromBody] BuildTransactionRequest request
                     }
                 }
 
+                var (opReturnRawData, isValid) = request.OpReturnDataIsHex ? TryGetHexValue(request.OpReturnData) : (null, true);
```

Comment:
If this is to be accepted, it must also be added to the "GetTransactionFeeEstimate" method or the fee calculation will be wrong and different compared to the actual transaction building.

Reply:
I'm fixing this myself, adding the same logic to the fee-calculation method.